crates/store/re_datafusion/src/dataframe_query_common.rs

```diff
@@ -190,6 +191,49 @@ impl DataframeQueryTableProvider {
             chunk_request,
         })
     }
+
+    fn column_to_selector(column: &Column) -> Option<ComponentColumnSelector> {
```

Comment:
Nit/style: we usually prefer `selector_from_column` (see [`CODE_STYLE.md`](https://github.com/rerun-io/rerun/blob/main/CODE_STYLE.md#matrices)).

Mostly for consistency with matrices, but also writing it `let selector = selector_from_column(column);` means the names in the function title is next to the types of the input/output variables (instead of being swizzled)